docs/themes/porter/assets/sass/helm-home.scss

```diff
@@ -55,14 +54,14 @@ nav.home-nav {
 
       a {
         display: inline-block;
-        margin: 0 1.333rem;
+        margin: 0 1rem;
         font-size: 1.25rem;
         line-height: 2;
         position: relative;
         letter-spacing: 0.0125em;
         color: $dark2;
         text-decoration: none;
-        @include klinicBold;
+        @include CBlack;
```

Comment:
Now that we are using this other font, can you do a sweep for the old klinic font, remove any references and remove the font files from the repository? I want to make sure that

a) our fonts are consistent and
b) we aren't accidentally having people download the unused font.